x-pack/plugins/fleet/public/applications/integrations/sections/epm/screens/detail/assets/deferred_transforms_accordion.tsx

```diff
@@ -0,0 +1,258 @@
+          throw reauthorizeTransformResp.error;
+        }
+        if (Array.isArray(reauthorizeTransformResp.data)) {
+          const error = reauthorizeTransformResp.data.find((d) => d.error)?.error;
+
+          const cntAuthorized = reauthorizeTransformResp.data.filter((d) => d.success).length;
+          if (error) {
+            const errorBody = error.meta?.body as ElasticsearchErrorDetails;
+            const errorMsg = errorBody
+              ? `${errorBody.error?.type}: ${errorBody.error?.reason}`
+              : `${error.message}`;
+
+            notifications.toasts.addError(
+              { name: errorMsg, message: errorMsg },
+              {
+                title: i18n.translate(
+                  'xpack.fleet.epm.packageDetails.assets.authorizeTransformsUnsuccessful',
+                  {
+                    defaultMessage:
+                      'Unable to authorize {cntUnauthorized, plural, one {# transform} other {# transforms}}.',
+                    values: { cntUnauthorized: transformIds.length - cntAuthorized },
+                  }
+                ),
+                toastLifeTimeMs: 1000,
+              }
+            );
+          } else {
+            notifications.toasts.addSuccess(
+              i18n.translate(
+                'xpack.fleet.epm.packageDetails.assets.authorizeTransformsSuccessful',
+                {
+                  defaultMessage:
+                    'Successfully authorized {count, plural, one {# transform} other {# transforms}}.',
+                  values: { count: cntAuthorized },
+                }
+              ),
+              { toastLifeTimeMs: 1000 }
+            );
+          }
+        }
+      } catch (e) {
+        if (e) {
+          notifications.toasts.addError(e, {
+            title: i18n.translate(
+              'xpack.fleet.epm.packageDetails.assets.unableToAuthorizeAllTransformsError',
+              {
+                defaultMessage: 'An error occurred authorizing and starting transforms.',
+              }
+            ),
+          });
+        }
+      }
+      setIsLoading(false);
+    },
+    [notifications.toasts, packageInfo.name, packageInfo.version]
+  );
+  if (deferredTransforms.length === 0) return null;
+  return (
+    <EuiAccordion
+      initialIsOpen={true}
+      buttonContent={
+        <EuiFlexGroup justifyContent="center" alignItems="center" gutterSize="s" responsive={false}>
+          <EuiFlexItem grow={false}>
+            <EuiText size="m">
+              <h3>{AssetTitleMap[type]}</h3>
+            </EuiText>
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiNotificationBadge color="accent" size="m">
+              <h3>{deferredTransforms.length}</h3>
+            </EuiNotificationBadge>
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      }
+      id={type}
+    >
+      <>
+        <EuiSpacer size="m" />
+
+        <EuiText>{getDeferredAssetDescription(type, deferredInstallations.length)} </EuiText>
```

Comment:
@qn895 @szabosteve shouldn't this message be dependent on the `enabled` state of the 'Reauthorize all' button? Here I have logged back in as a user (`elastic`) with the required privileges but the text is still telling me to contact the administrator. Maybe something along the lines of `The transform was installed by a user who did not have the permissions required to run it. Click the button below to reauthorize the transform.`

<img width="831" alt="image" src="https://user-images.githubusercontent.com/7405507/233061290-e0b647f4-0163-4551-8994-528710813e4d.png">

Reply:
@peteharverson @qn895 
Thanks for the info! 
Sorry for the confusion, I had a wrong assumption about the conditions the message is shown.
If I understand correctly and the message is dependent on the state of the Reauthorize button, then we have two scenarios:
1. The Reauthorize button is NOT clickable,
2. The Reauthorize button is clickable.

If my assumption is correct this time, we can use the current text for the first scenario (the user doesn't have proper permissions). In the second case (the user has the proper permissions, so the button is enabled), we can use something like: `The transform was installed by a user who did not have the permissions required to run it. Reauthorize the transform by clicking the button below.` Which is a slightly modified version of what Pete suggested.

Please let me know if I still miss anything or if we need a message that is not dependent on the state of the 'Reauthorize' button.